### PR TITLE
Release

### PR DESCRIPTION
**Pull request by vireshk**
Move virtio-queue, virtio-bindings and virtio-vsock to newer versions.

### CODE REVIEW

File: crates/devices/virtio-vsock/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "virtio-vsock"
-version = "0.3.0"
+version = "0.4.0"
```
Comment (jiangliu) [inline crates/devices/virtio-vsock/Cargo.toml:3]: Seem there's no functional/interface change, so should it be "0.4.0" or "0.3.1"?

File: crates/virtio-bindings/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "virtio-bindings"
-version = "0.2.0"
+version = "0.3.0"
```
Comment (jiangliu) [inline crates/virtio-bindings/Cargo.toml:3]: 0.2.1?

Comment (vireshk) [reply crates/virtio-bindings/Cargo.toml]: Fixed both. Thanks.

**Comment by vireshk** [comment]
@jiangliu Any idea where the `buildkite/fuzzing-vm-virtio` is stuck ? Opening the page gives me "page not found" error.

**Comment by jiangliu** [comment]
> @jiangliu Any idea where the `buildkite/fuzzing-vm-virtio` is stuck ? Opening the page gives me "page not found" error.

I'm not sure, does a `force puhsing` help?

**Comment by vireshk** [comment]
> I'm not sure, does a `force puhsing` help?

I see this issue on all the currently open pull requests to vm-virtio, don't think this is just about my request.

**Comment by vireshk** [comment]
and blocked again.

**Review by stsquad — APPROVED** [review]

**Review by jiangliu — APPROVED** [review]

**Review by stsquad — APPROVED** [review]

**Pull request merged**